### PR TITLE
[KotlinCleanup] use kotlin.test.* for asserting

### PR DESCRIPTION
**Pull request by JalinWang**
https://github.com/ankidroid/Anki-Android/issues/10489

### CODE REVIEW

File: api/src/test/java/com/ichi2/anki/api/ApiUtilsTest.kt [hunk 1]
```diff
@@ -75,8 +74,4 @@ internal class ApiUtilsTest {
     fun shouldGenerateProperCheckSum() {
         assertEquals(3533307532L, Utils.fieldChecksum("AnkiDroid"))
     }
-
-    companion object {
-        private const val delimiter = "\u001F"
```
Comment (david-allison) [inline api/src/test/java/com/ichi2/anki/api/ApiUtilsTest.kt:80]: This should be kept (likely with a comment added). 

A change to `FIELD_SEPARATOR` should break tests

**Review by david-allison — CHANGES_REQUESTED** [review]
Looks great! One Request

File: api/src/test/java/com/ichi2/anki/api/ApiUtilsTest.kt (hunk 1 above)
Comment (JalinWang) [reply api/src/test/java/com/ichi2/anki/api/ApiUtilsTest.kt:80]: uhh, I see. Confusing one🤣.

**Review by david-allison — APPROVED** [review]

**Review by BrayanDSO — APPROVED** [review]
LGTM

**Pull request merged**